src/openfermion/functionals/get_one_norm_test.py

```diff
@@ -0,0 +1,49 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Tests for get_one_norm."""
+
+import os
+import unittest
+from openfermion import get_one_norm, MolecularData, jordan_wigner
+from openfermion.config import DATA_DIRECTORY
+
+class test_get_one_norm(unittest.TestCase):
```

Comment:
Nit: I think we're moving slightly away from unittest and towards pytest? I have no personal preference here, just wanting to flag.

Reply:
I'm fine with everything :) I just mimicked some other test scripts

Reply:
Yes please move over to pytest